packages/animations/lib/src/utils/curves.dart

```diff
@@ -49,8 +49,8 @@ class FlippedCurveTween extends CurveTween {
   /// Creates a vertically flipped [CurveTween].
   FlippedCurveTween({
     @required Curve curve,
-  }) : assert(curve != null),
-       super(curve: curve);
+  })  : assert(curve != null),
```

Comment:
extra spaces ended up here

Reply:
What's weird is that the formatter suggested two spaces instead of one before the colon `:`. If not, the formatted complained and I'm unsure if that's a normal expectation for non-`flutter/flutter` projects or not

Reply:
The Dart formatter doesn't really do conventional Flutter SDK formatting.